src/__tests__/deconv-examples/data_example1.js

```diff
@@ -0,0 +1,1133 @@
+export const X = [
```

Comment:
Could you add a test with all the values (in X and Y) - 1. Adding 1 was a way to circumvent previous bug.